src/screens/WaitTimeScreen.tsx

```diff
@@ -360,7 +361,7 @@ const styles = StyleSheet.create({
     borderTopRightRadius: 10,
     paddingBottom: 10,
     paddingHorizontal: 20,
-    paddingTop: 20,
+    paddingTop: 5,
```

Comment:
I think 5 is too small, 10 seems like a good middle

Reply:
Feel free to push the change tonight, or I can do it tmmrw.